### PR TITLE
APEXMALHAR-2061 fixing travis build failures

### PR DESCRIPTION
**Pull request by yogidevendra**
1. Restricting memory for surefire plugin

### CODE REVIEW

File: samples/pom.xml [hunk 1]
```diff
@@ -33,7 +33,18 @@
   <packaging>jar</packaging>
 
   <name>Apache Apex Malhar (incubating) Samples</name>
-
+  <build>
```
Comment (tweise) [inline samples/pom.xml:36]: See if you can use the property checkstyle.skip instead.

**Comment by tweise** [comment]
Would suggest to combine all checkstyle related changes into single commit.

File: pom.xml [hunk 2]
```diff
@@ -75,6 +75,13 @@
         </configuration>
       </plugin>
       <plugin>
+      	<groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
```
Comment (siyuanh) [inline pom.xml:80]: In one line?

Comment (yogidevendra) [inline pom.xml:80]: @siyuanh 
Is this a general comment or review comment?
Honestly, I did not get your feedback about what needs to be changed.

**Comment by yogidevendra** [comment]
Incorporated review comments.
Please merge.

File: samples/pom.xml (hunk 1 above)
Comment (tweise) [inline samples/pom.xml:36]: Adding "checkstyle.skip" under properties does not work?

https://maven.apache.org/plugins/maven-checkstyle-plugin/check-mojo.html

File: pom.xml [hunk 3]
```diff
@@ -75,6 +75,13 @@
...
+    		<argLine>-Xmx512m</argLine>
+  		</configuration>
```
Comment (siyuanh) [inline pom.xml:82]: Sorry, wrong comment, but the indentation here is not correct?

Comment (tweise) [inline pom.xml:82]: Yes, formatting wasn't correct. Next PR will fix it. @yogidevendra please check your editor settings.

**Pull request merged**